nodejs/src/test-helper.mjs

```diff
@@ -222,6 +222,72 @@ export default function testSuite(
 
 			expect(stream).toBeInstanceOf(ReadableStream);
 		});
+
+		test("handles audio-server stream errors correctly", async () => {
+			const mockStreamResponse = new Response(
+				new ReadableStream({
+					async pull(controller) {
+						controller.enqueue(new Uint8Array([1, 2, 3]));
+						await new Promise((resolve) => setTimeout(resolve, 1000));
+						controller.enqueue(new Uint8Array([4, 5, 6]));
+						await new Promise((resolve) => setTimeout(resolve, 1000));
+						controller.error(new Error("Some error from audio server"));
+						controller.enqueue(new Uint8Array([7, 8, 9]));
+					},
+				}),
+			);
+
+			globalThis.fetch = vi.fn();
```

Comment:
This should be done in test setup and cleaned in the teardown
https://vitest.dev/guide/mocking.html#example